setup.py

```diff
@@ -93,12 +98,34 @@ def run(self):
             )
 
 
+def get_version(version: "ScmVersion") -> str:
+    from setuptools_scm import version as scm_version
+
+    if version.node:
+        version.node = str(int(version.node.lstrip("g"), 16))
```

Comment:
Where's the g coming from (G for git?)

Reply:
Yes, it appends "g" before the first few characters of the commit hash if it comes from git (and does something else if using mercurial, ugh).